Dockerfile

```diff
@@ -1,12 +1,28 @@
-FROM node:20-slim
-RUN apt update
-RUN apt upgrade --yes
+FROM node:alpine as builder
 
-# Necessary for /update
-RUN apt install git --yes
+WORKDIR /app
 
-# Necessary to install devDependencies
-ENV NODE_ENV=development
+COPY . .
 
-WORKDIR /cs-bot
-ENTRYPOINT [ "bash" ]
+RUN npm ci
+RUN npm run export-version
+RUN npm run build --omit=dev
+
+FROM node:alpine as runner
+
+WORKDIR /app
+
+COPY --from=builder /app/dist/ ./dist/
+COPY --from=builder /app/res/ ./res/
+COPY package*.json ./
+COPY prisma prisma/
+COPY scripts/launch_in_docker.sh .
+
+RUN npm ci --omit=dev
+
+# Path internal to container; use `volumes` config to specify real system path of `/db/`
+ENV DATABASE_URL="file:/db/db.sqlite"
+
+RUN npm run db:migrate
```

Comment:
Good catch! I guess we should run migrations in `launch_in_docker.sh` too, instead of here (which I think happens on _build_). I'll push a fix for this and the image in a few mins

Reply:
Updated, and edited the PR description with that note.